style.go

```diff
@@ -183,7 +183,8 @@ func DefaultFill() *Fill {
 }
 
 func DefaultBorder() *Border {
-	return NewBorder("none", "none", "none", "none")
+	//return NewBorder("none", "none", "none", "none")
+	return NewBorder("", "", "", "")
```

Comment:
Thanks for the explanation, however I'm not quite sure what the takeaway here was, can we revert this change or should we remove the commented code? :D